### PR TITLE
Added Nahmii Testnet

### PR DESCRIPTION
**Pull request by jonddon**
Added Nahmii Testnet

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -574,5 +574,16 @@
     "multicall": "0x4617D470F847Ce166019d19a7944049ebB017400",
     "rpc": ["https://crab-rpc.darwinia.network"],
     "explorer": "https://crab.subscan.io/"
+  },
+    "5553": {
```
Comment (ChaituVR) [inline src/networks.json:578]:
```suggestion
  },
  "5553": {
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Works good @bonustrack 

image

**Comment by bonustrack** [comment]
@jonddon The image should be a square, can you change it?

**Comment by jonddon** [comment]
> 

I have changed the logo

**Comment by jonddon** [comment]
> @jonddon The image should be a square, can you change it?

I have updated the logo

**Pull request closed without merging**